pkg/trace/sampler/prioritysampler_test.go

```diff
@@ -6,17 +6,14 @@
 package sampler
 
 import (
+	"go.uber.org/atomic"
```

Comment:
I think this import list needs to be sorted

Reply:
Fixed :)

(how did you catch that omg)

Reply:
🦅 👀